lib/funky/video.rb

```diff
@@ -45,6 +45,15 @@ def page_name
       data.fetch(:from)[:name]
     end
 
+    # @return [String] the id of Facebook page for the video.
+    def page_id
+      data.fetch(:from)[:id]
+    end
+
+    def page_url
+      "https://www.facebook.com/" + page_id
```

Comment:
@bruce-fs  Interpolation might be better performing than concatenation.

See https://github.com/johnotander/ruby_optimization_techniques#45-simple-code-enhancements